device/deviceType/deviceType.go

```diff
@@ -0,0 +1,9 @@
+package deviceType
+
+type Type string
+
+const (
```

Comment:
If this just contains some consts, I'm not sure why this is a separate package? I'd put this in `device`.